config/deploy.rb

```diff
@@ -105,3 +105,34 @@
     end
   end
 end
+
+namespace :application do
+  # You can/ should apply this command to a single host
+  # cap --hosts=dpul-staging3.princeton.edu staging application:remove_from_nginx
+  desc "Marks the server(s) to be removed from the loadbalancer"
+  task :remove_from_nginx do
+    count = 0
+    on roles(:app) do
+      count += 1
+    end
+    if count > 1
```

Comment:
Can we do the check we did here? https://github.com/pulibrary/figgy/pull/6539/files#diff-3d066bc83eaf220ed7d8276202a78415dfe66ddf285f3382273b86d5845ab9c4R92